web/src/assets/blank-canvas.svg

```diff
@@ -0,0 +1,120 @@
+<svg width="415" height="346" viewBox="0 0 415 346" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this svg only for merchant? if yes can we create a sub folder 'merchant'

Reply:
@ifhuang okay!